src/longbow/segment/command.py

```diff
@@ -647,6 +672,13 @@ def create_simple_split_array_element(delim_name, end_coord, model, prev_delim_n
 
             a.set_tag(bam_utils.READ_BARCODE_CONF_FACTOR_TAG, conf_factor)
 
+    # Set IsoSeq3-compatible tags:
+    a.set_tag(bam_utils.READ_CLIPPED_SEQS_LIST_TAG, ','.join(clipped_tags))
+    a.set_tag(bam_utils.READ_NUM_CONSENSUS_PASSES_TAG, 1)
+    a.set_tag(bam_utils.READ_ZMW_NAMES_TAG, read.query_name)
+    a.set_tag(bam_utils.READ_NUM_ZMWS_TAG, 1)
+    a.set_tag(bam_utils.READ_TAGS_ORDER_TAG, f'{bam_utils.READ_RAW_BARCODE_TAG}-{bam_utils.READ_RAW_UMI_TAG}')
```

Comment:
This is fine for now, but I think with the SlideSeq data some of this might be a problem since it has two spatial barcodes.

Reply:
Agreed